src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -973,6 +973,14 @@ public override BoundNode VisitLambda(BoundLambda node)
             return null;
         }
 
+        public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatement node)
+        {
+            if (_trackExceptions) NotePossibleException(node);
```

Comment:
You don't need this line. A local function declaration cannot throw an exception due to normal language rules.